README.md

```diff
@@ -4,3 +4,207 @@
 
 `ppx_factory` is an OCaml preprocessor to derive factory methods and default values from type
 definitions.
+
+## Overview
+
+Factories are functions that let you build test data while only specifying the parts that are
+relevant to your tests.
+
+`ppx_factory` alows you to derive such functions from type definitions.
```

Comment:
Is it worth mentioning that the `default` part of the PPX exists here as well?

Reply:
Yeah it might be, I'll try to add something!

Reply:
I added a mention of `default`, let me know what you think!

Reply:
Yeah thats helpful thanks